pygenomeworks/genomeworks/cudaaligner/cudaaligner.pyx

```diff
@@ -129,18 +130,18 @@ cdef class CudaAlignerBatch:
     """Python API for CUDA-accelerated sequence to sequence alignment.
     """
     cdef unique_ptr[cudaaligner.Aligner] aligner
-    cdef public object stream
+    cdef public CudaStream stream
 
     def __cinit__(
             self,
             max_query_length,
             max_target_length,
             max_alignments,
+            *args,
```

Comment:
@tijyojwad Having `*args` as an argument before any key argument can lead to a bug. This way, the key-arguments after will only be able to be set by explicitly setting them when calling the function, since all the remaining positional arguments will be collected by `*args` parameter.
For instance:
```
def foo(a, b, *args, c=-1, **kargs):
    print(args)
    print(c)
    print(kargs)


if __name__ == '__main__':
    foo(1, 2, 3, d=5, e=8)
```
Output
```
(3,)
-1
{'d': 5, 'e': 8}
```
Compared to:
```
def foo(a, b, c=-1, *args, **kargs):
    print(args)
    print(c)
    print(kargs)


if __name__ == '__main__':
    foo(1, 2, 3, d=5, e=8)
```
Ouput:
```
()
3
{'d': 5, 'e': 8}
```

Reply:
Good catch @o8191 ! Fixed. Can you review?